### ISSUE TITLE
[Tests] Implement stress testing and benchmarking

### ISSUE THREAD

**Issue by benjaminbollen** [hyperledger/burrow#212]
Stress tests need to be designed at unit test level and at network tests level.  Stress testing must happen along two dimensions: intensity and duration.  The aim is deliberately to expose and map out the breaking points of the stack.

**Comment by compleatang** [comment]
ensure that, as a good open source project, we are publishing these results.

**Comment by compleatang** [comment]
this issue is largely finalized with the work against caliper and our internal stress tests. although we are blocked by the implementation of a hyperledger CI system for operating stress networks I think we should close this issue for now. 

see: https://lists.hyperledger.org/g/tsc/topic/30876684 for future work.
